src/applications/vaos/appointment-list/redux/actions.js

```diff
@@ -432,12 +437,36 @@ export function fetchExpressCareWindows() {
 
     const initialState = getState();
     const userSiteIds = selectSystemIds(initialState);
+    const address = selectVet360ResidentialAddress(initialState);
 
     try {
       const settings = await getRequestEligibilityCriteria(userSiteIds);
+      let facilityData;
+
+      if (address?.latitude && address?.longitude) {
```

Comment:
I think we'll also need to check if the address is a PO Box and skip this if so.  
The mock data residential address in `local-vet360.js` is a PO box, and I think there are some other tests elsewhere in the app that rely on that address. 

This is the regex we're using on the TypeOfCarePage alert: `addressLine1.match(/^PO Box/)`

Reply:
Yeah, I was trying to think through if we need to do any filtering based on the type of address it is. I was leaning towards no, because even a lat/long on a PO Box could be useful if it's near your current location, and if it isn't, it's not necessarily any more misleading than our current arbitrary choice of the first facility.

Reply:
Good point. Ignoring the PO box scenario sounds fine for now.